jobs/gorouter/spec

```diff
@@ -438,6 +438,13 @@ properties:
       When it set to true, the gorouter will return a 503 when a route is known but there is nothing in the pool available to handle the request. When false, the gorouter will return a 404.
     default: true
 
+  for_backwards_compatibility_only.empty_pool_timeout:
+    description: |
+      This property depends on empty_pool_response_code_503. if empty_pool_response_code_503 is set to true, then gorouter will consider empty_pool_timout parameter.
+      If set to 0, gorouter will remove empty pools after pruning their endpoints. The gorouter will respond with 404 code.
+      If set to different value, gorouter will wait until this timeout is reached then will delete the empty pools. Before pool deletion, gorouter will continue to respond with 503 code.
+    default: 10
```

Comment:
The value is parsed as `time.Duration` in [the PR](https://github.com/cloudfoundry/gorouter/pull/314/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R291) so putting a default of `10` actually means `10ns` (the internal value of time.Duration).

Since we want 0 to be the default, you could either put `0s` as the default or put an example like
```
example: 10s
```
next to the default, so the people will know this is not an integer value but a duration string.

Maybe even add both?

Reply:
Good point. I've updated that and added some explanation about the type of this parameter.